scripts/channel_map.py

```diff
@@ -3,8 +3,8 @@
 class ChannelMap():
     channel_map = {
         'main': {
-            'tfm': 'net8.0',
-            'branch': '8.0',
+            'tfm': 'net7.0',
```

Comment:
Is this used only to decide which tfm to use on `dotnet/runtime` `main` branch?

Reply:
yes, that is correct